dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -41,6 +41,15 @@ update_default_gw()
    fi
 }
 
+write_default_zebra_config()
+{
+    FILE_NAME=${1}
+
+    grep -q '^no fpm use-next-hop-groups' $FILE_NAME || {
+        sed -i '1i no fpm use-next-hop-groups\nfpm address 127.0.0.1' $FILE_NAME
```

Comment:
Please refer to the comment by @dgsudharsan https://github.com/sonic-net/sonic-buildimage/pull/12852#discussion_r1067747770.
SONiC supports split mode FRR configuration. That is, user provides own zebra.conf, bgp.conf, etc. in /etc/sonic/frr folder, instead of generated from templates by SONiC.
In case user upgrades from older version to image with these changes he will not have mandatory configuration in his zebra.conf copied from the old image. I added this line to insert this configuration in case user does not have it.